src/manual/peer2peer/js/main.js

```diff
@@ -411,7 +411,12 @@ function screenCaptureExtensionHandler_() {
     // user chose a stream
     if (event.data.type && (event.data.type === 'SS_DIALOG_SUCCESS')) {
       var constraints = {
-        audio: false,
+        audio: {
+          mandatory: {
+            chromeMediaSource: 'desktop',
```

Comment:
I've tested, and it can work (although one cannot really get audio stream, chrome does not return an error for this.)
If you like, I can add a version check for this, too.